grammars/python.cson

```diff
@@ -238,13 +238,17 @@
     'beginCaptures':
       '1':
         'name': 'storage.type.function.python'
-    'end': '(\\))\\s*(?:(\\:)|(.*$\\n?))'
+    'end': '(\\))\\s*(?:(->)\\s*([A-Za-z_][A-Za-z0-9_]*)\\s*)?(?:(\\:)|(.*$\\n?))'
     'endCaptures':
       '1':
         'name': 'punctuation.definition.parameters.end.python'
       '2':
-        'name': 'punctuation.section.function.begin.python'
+        'name': 'punctuation.definition.annotation.return.python'
       '3':
+        'name': 'variable.annotation.function.python'
```

Comment:
From [TextMate docs:](http://manual.macromates.com/en/language_grammars#naming_conventions)

>
variable — variables. Not all languages allow easy identification (and thus markup) of these.
- parameter — when the variable is declared as the parameter.
- language — reserved language variables like this, super, self, etc.
- other — other variables, like $some_variables.